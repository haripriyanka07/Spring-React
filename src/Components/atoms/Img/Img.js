import "../Img/Img.css";

const Img = (props) => {
    return (
        <img  src={props.src} alt={props.alt} />
    )
}
export default Img;