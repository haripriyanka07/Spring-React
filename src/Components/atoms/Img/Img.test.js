import React from 'react';
import Img from './Img';
import renderer from 'react-test-renderer';

it('test the img', () => {
    const asFragment = renderer.create(<Img src="hari" alt="hari img" />).toJSON();
    expect(asFragment).toMatchSnapshot()
})

