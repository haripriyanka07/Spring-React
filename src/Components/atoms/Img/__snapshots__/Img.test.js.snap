// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the img 1`] = `
<img
  alt="hari img"
  src="hari"
/>
`;
